spec/controllers/catalog_controller_spec.rb

```diff
@@ -54,6 +54,53 @@
       end
     end
   end
+  describe 'POST submit_cart_updates_form on item' do
+    before(:each) do
+      @equipment_model = FactoryGirl.create(:equipment_model)
+      put :add_to_cart, id: @equipment_model.id
+    end
+    it 'should adjust item quantity' do
+      # check if cart contains an item
+      expect(session[:cart].items).not_to be_empty
```

Comment:
Should this come after the POST request on line [68](https://github.com/YaleSTC/reservations/pull/1221/files#diff-66e8d5275f70f4e66a363591449594e7R68) below?